tutorial/extending-widgets.md

```diff
@@ -68,3 +68,91 @@ func main() {
 	w.ShowAndRun()
 }
 ```
+
```

Comment:
To match the heading at the top, how about adding:

```
## Extending a Widget Renderer
```

or similar?
At some point we may rearrange the tutorials - this makes it clear it's a topic change